assets/js/main.js

```diff
@@ -91,6 +93,20 @@ const projects = [
 // add projects dynamically
 
 window.addEventListener('DOMContentLoaded', () => {
+  // get local storage form data
+  if (localStorage.getItem('user') === null) {
+    email.value = '';
+    message.value = '';
+    userName.value = '';
+  } else {
+    const localData = localStorage.getItem('user');
+    const data = JSON.parse(localData);
+
+    // set localstorage data to the input values
+    email.value = data.email;
+    message.value = data.message;
+    userName.value = data.name;
+  }
```

Comment:
- [ ] Kindly make sure to preserve the form data in local storage. You're currently checking if the local storage has data and populating the fields with that data. However, you're not saving the data or setting anything in the local storage if there's nothing already there. 
Please make sure to add this functionality. (You can use the `onChange` event to get data from the forum and save them in the local storage with every user input)